Resources/Markdown/faq.md

```diff
@@ -146,50 +146,52 @@ If you don't have the SPI-Playgrounds app installed, you can [download it for ma
 
-For example, a query of [`charts last_activity:>=2021-02-01`](https://swiftpackageindex.com/search?query=charts+last_activity%3A%3E2021-01-01) matches packages containing the word "charts" and that have also received some activity since the start of February 2021.
+#### Querying License values
 
-#### Querying licenses
+You can search for packages that have a license which is [compatible with the App Store](#licenses) by simply querying [`license:compatible`](https://swiftpackageindex.com/search?query=license%3Acompatible).
 
-You can search for packages that have a license which is compatible with the App Store by simply querying [`license:compatible`](https://swiftpackageindex.com/search?query=license%3Acompatible).
+You can also specify a specific license you wish a package to have from one of the [built-in options](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/blob/main/Sources/App/Models/License.swift). For example, a query of [`license:lgpl-2.1`](https://swiftpackageindex.com/search?query=license%3Algpl-2.1) matches any package licensed under the LGPL 2.1.
 
-You can also specify a specific license you wish a package to have from one of our [built-in options](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/blob/main/Sources/App/Models/License.swift). For example, a query of [`license:lgpl-2.1`](https://swiftpackageindex.com/search?query=license%3Algpl-2.1) matches any package with the license LGPL 2.1.
+#### Querying Author or Keyword values
 
-#### Querying authors or keywords
-
-You can also search for packages created by a specific user or organisation by filtering on `author`. For example, a query of [`fluent author:vapor`](https://swiftpackageindex.com/search?query=fluent+author%3Avapor) matches any package containing the word "fluent" owned by the vapor organisation.
+You can search for packages created by a specific user or organisation by filtering on `author`. For example, a query of [`fluent author:vapor`](https://swiftpackageindex.com/search?query=fluent+author%3Avapor) matches any package containing the word "fluent" owned by the vapor organisation.
 
 Similarly, you can search for packages that contain keywords with a `keyword` filter. For example, [`keyword:accessibility`](https://swiftpackageindex.com/search?query=keyword%3Aaccessibility) matches any package having the "accessibility" keyword.
 
-#### Excluding results
+#### Querying Platform values
+
+You can search for packages that are compatible with a platform using a `platform` filter. Specify multiple platforms together with commas. For example, [`layout platform:ios,linux`](https://swiftpackageindex.com/search?query=layout+platform%3Aios%2Clinux) shows packages matching the word "layout" that are compatible with both iOS and Linux.
```

Comment:
This query clearly does work, but I wonder if it's the best example we could give? AutoLayout for Linux is an interesting concept 😅  I'm not sure the results we show are the best here.

Reply:
I'd be open to other queries as long as they don't repeat the same word as other examples, but the query isn't AutoLayout, it's "layout".

Reply:
I understand it's looking for "layout" and not "autolayout" but the results are mostly around AutoLayout which doesn't make a tonne of sense given the use cases. As I say, it's not wrong, just maybe not the best.

> don't repeat the same word as other examples

I was very intentional with that in my other examples 😛  Tried to give a variety of examples.

Reply:
Yep, I saw that and appreciated it. Any suggestions on what to use here instead of "layout"? I don't see it as a huge issue, but if you want to change it then you need to be the one suggesting a replacement 😂

Reply:
Perhaps "testing" for testing libraries which work on both iOS and Linux?

https://swiftpackageindex.com/search?query=testing+platform%3Aios%2Clinux